library/core/src/slice/iter.rs

```diff
@@ -304,6 +304,47 @@ impl<'a, T> IterMut<'a, T> {
     pub fn as_slice(&self) -> &[T] {
         self.make_slice()
     }
+
+    /// Views the underlying data as a mutable subslice of the original data.
+    ///
+    /// To avoid creating `&mut [T]` references that alias, the returned slice
+    /// borrows its lifetime from the iterator the method is applied on.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # #![feature(slice_iter_mut_as_mut_slice)]
```

Comment:
We usually show the unstable features in examples:
```suggestion
    /// #![feature(slice_iter_mut_as_mut_slice)]
```